metadata.rb

```diff
@@ -6,7 +6,7 @@
 chef_version     '>= 14'
 license          'Apache-2.0'
 description      'Installs and configures nginx'
-version          '10.3.2'
+version          '10.3.3'
```

Comment:
Shouldnt change the version the release automation will adjust it automatically.
```suggestion
version          '10.3.2'
```